angular/projects/spark-angular/src/lib/components/sprk-flag/sprk-flag.component.spec.ts

```diff
@@ -24,22 +24,36 @@ describe('SprkFlagComponent', () => {
 
   it('should add the correct classes if additionalClasses are supplied', () => {
     component.additionalClasses = 'sprk-u-pam sprk-u-man';
-    expect(component.getClasses()).toEqual(
-      'sprk-o-Flag sprk-u-pam sprk-u-man'
+    expect(component.getClasses()).toEqual('sprk-o-Flag sprk-u-pam sprk-u-man');
+  });
+
+  it('should add the correct classes if mediaAdditionalClasses are supplied', () => {
+    component.mediaAdditionalClasses = 'sprk-u-pam sprk-u-man';
+    expect(component.getMediaClasses()).toEqual(
+      'sprk-o-Flag__figure sprk-u-pam sprk-u-man',
+    );
+  });
+
+  it('should add the correct classes if bodyAdditionalClasses are supplied', () => {
+    component.bodyAdditionalClasses = 'sprk-u-pam sprk-u-man';
+    expect(component.getBodyClasses()).toEqual(
+      'sprk-o-Flag__body sprk-u-pam sprk-u-man',
     );
   });
 
+  // TO DEPRECATE NEXT RELEASE
```

Comment:
I think we should use TODO in these two spots. That's what we use in other places and it will be easier to search for if we're consistent.

Reply:
![image](https://user-images.githubusercontent.com/4342363/89425117-99e63d00-d706-11ea-8183-01c37da8e386.png)
Done!